src/foam/u2/borders/CollapseBorder.js

```diff
@@ -32,42 +29,34 @@ foam.CLASS({
     ^toolbar {
       display: flex;
       flex-direction: row;
+      justify-content: space-between;
       align-items: center;
-      position: absolute;
-      padding: 3px;
-      width: calc(100% - 3vmin);
       left: 1.5vmin;
       top: max(-12px, -2vmin);
       color: #666;
-      padding-top: 0;
-    }
-    ^title {
-      background: /*%WHITE%*/ #ffffff;
-      position: relative;
+      padding: 0.8rem 0;
+      cursor: pointer;
     }
     ^toggle-button {
       font-size: inherit !important;
       border: none;
       outline: none;
-      padding: 3px;
     }
     ^toggle-button:focus {
       background-color: /*%WHITE%*/ #ffffff;
     }
-    ^control > ^toggle-button {
-      transform: rotate(-90deg);
+    ^control {
       transition: transform 0.3s;
-      width: 30px;
     }
-    ^.expanded ^toggle-button {
-      transform: rotate(0deg);
-      transition: transform 0.3s;
+    ^.expanded ^control {
+      transform: rotate(90deg);
     }
-    ^place-right {
-      order: 1;
+    ^control svg {
+      max-height: 14px;
+      fill: /*%BLACK%*/ #000;
     }
-    ^space-even {
-      justify-content: space-between;
+    ^content {
+      padding-bottom: 0.8rem;
```

Comment:
Shouldn't this be on the border? content might have it's own padding that would override this one